src/libraries/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -85,146 +85,6 @@ internal static string QueryContextAuthenticationPackage(SafeDeleteContext secur
             }
         }
 
-        private static bool GssInitSecurityContext(
```

Comment:
The `GssInitSecurityContext` and `GssAcceptSecurityContext` methods are folded into the caller. This made updating the handles substantially easier. I also removed the code that threw `GssApiException` only to catch it one method above and convert it to reported status code.